src/functions/functionVersionBump.ts

```diff
@@ -0,0 +1,108 @@
+import fs from 'fs-extra';
+import path from 'path';
+import prompts from 'prompts';
+
+import { functionDefinitions } from './functionDefinitions';
+
+const workingDir = process.cwd();
+const baseFunctionsPath = path.join(workingDir, 'functions');
+
+export const collectBumpInput = async (): Promise<{
```

Comment:
This function does multiple things, can you maybe divide it like this:

export const collectBumpInput {
    getFunctionDir()
    getFunctions()
    createDirName()
    prepareVersionedFunctions()
  .... etc
}

Reply:
fixed